anaUltraScurve.py

```diff
@@ -249,9 +321,8 @@ def overlay_fit(VFAT, CHAN):
             noise[0] = param1
             fitENC.append(vToQm*param1*options.ztrim)
             pedestal[0] = param2
-            if noise[0] > 20.0 or ped_eff[0] > 50.0: mask[0] = True
```

Comment:
We don't need to mask noisy channels if their operational threshold (`threshold-zTrim*noise`) is high. The MAD algorithm will detect low operational thresholds.

There is no replacement for the effective pedestal cut. I only saw S-curve plots without a white band at the bottom (ie bins with zero events) in untrimmed data after the HV incident, and most of them weren't masked by `anaUltraScurve`. Re-adding the cut wouldn't hurt, and in this case I'd even make it tighter.

(Note: changes to the trimming procedure may affect the second point, but I'd say that if trimming results in channels having to be masked, then the trimming procedure is bad.)